<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>baseInfo</class>
 <widget class="QWidget" name="baseInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>858</width>
    <height>508</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>858</width>
    <height>508</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>学号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>班级</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>照片</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sexLabel">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ageLabel">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="numLabel">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="classLabel">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
